src/app/Modules/error/page-error/page-error.component.css

```diff
@@ -0,0 +1,3 @@
+div{
```

Comment:
Check why prettier didn't work on the files with this kind of style error, will only leave the comment on this one but there are more in the other files.